plugins/mongodb/src/main/java/com/navercorp/pinpoint/plugin/mongo/MongoUtil.java

```diff
@@ -86,5 +95,66 @@ public static NormalizedBson parseBson(Object[] args, boolean traceBsonBindValue
         String jsonParameterString = StringJoiner.join(jsonParameter, SEPARATOR);
         return new NormalizedBson(parsedJsonString, jsonParameterString);
     }
+
+    public static NormalizedBson parseJson(Object[] args, boolean traceJsonBindValue) {
+        StringBuilder parsedJsonBuilder = new StringBuilder();
+        StringBuilder jsonParamBuilder = new StringBuilder();
+        parsedJsonBuilder.append("{");
+        for (Object arg : args) {
+            if (arg instanceof DBObject[]) {
+                DBObject[] objects = (DBObject[]) arg;
+                for (DBObject object : objects) {
+                    extractFromDbObject(parsedJsonBuilder, jsonParamBuilder, object);
+                }
+                adjustJson(parsedJsonBuilder, jsonParamBuilder);
+            } else if (arg instanceof DBObject) {
+                extractFromDbObject(parsedJsonBuilder, jsonParamBuilder, (DBObject) arg);
+                adjustJson(parsedJsonBuilder, jsonParamBuilder);
+            } else {
+                System.err.println("Not a DBObject; it's a(n) " + arg.getClass());
+            }
+        }
+
+        String parsedJsonString = parsedJsonBuilder.toString();
+        String jsonParameterString = jsonParamBuilder.toString();
+        System.out.println("parsedJsonString - " + parsedJsonString + "; jsonParameterString - " + jsonParameterString);
+        return new NormalizedBson(parsedJsonString, jsonParameterString);
+    }
+
+    /**
+     * @param parsedJsonBuilder
+     * @param jsonParamBuilder
+     * @param object
+     */
+    private static void extractFromDbObject(StringBuilder parsedJsonBuilder, StringBuilder jsonParamBuilder,
+            DBObject object) {
+        for (String key : object.keySet()) {
+            if (key.equals("_id")) {
+                continue;
+            }
+            parsedJsonBuilder.append("\"");
+            jsonParamBuilder.append("\"");
+            parsedJsonBuilder.append(key);
+            parsedJsonBuilder.append("\"");
+            parsedJsonBuilder.append(":");
+            parsedJsonBuilder.append("\"");
+            parsedJsonBuilder.append("?");
+            jsonParamBuilder.append(object.get(key));
+            parsedJsonBuilder.append("\"");
+            jsonParamBuilder.append("\"");
+            parsedJsonBuilder.append(",");
```

Comment:
if this ends with `,` 

doesn't the String be something like "a,b,c," ?

Please check once again

Reply:
I can see the last "," is taken care of from adjustJson method.
But I think this is an unnecessary procedure. What do you think?
I think it would be better not to add last "," from the first place.